ext/symengine/symengine.h

```diff
@@ -11,5 +11,6 @@ VALUE c_basic;
 VALUE c_symbol;
 VALUE c_integer;
 VALUE c_rational;
+VALUE c_constants;
```

Comment:
Can you assign to this variable in `symengine.c`?
Also you have to add it to `Klass_of_basic` method